### DISCUSSION TITLE
Performance issue with coreshop price

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [coreshop/CoreShop#2343] [Q&A]
Hi @dpfaffenbauer 
We use the Coreshop in one of our projects, all things are done perfectly with Coreshop but now we face the issue with performance.
during the debug code i found one twig function related to price which is taking too much time.

function: product_price.display_product_price

file location: \vendor\coreshop\core-shop\src\CoreShop\Bundle\FrontendBundle\Resources\views\Common\Macro\product_price.html.twig

Function calling from:  \vendor\coreshop\core-shop\src\CoreShop\Bundle\FrontendBundle\Resources\views\Product\detail.html.twig
line:84 -> product_price.display_product_price(product)

Can you please guide me, on what needs to be done here?

**Comment by dpfaffenbauer** [comment]
A blackfire profile would be very helpful.

**Reply by dpfaffenbauer** [reply]
@rushitkhunt1996 any update?

**Reply by dpfaffenbauer** [reply]
ping @rushitkhunt1996
